EmbedFactory.py

```diff
@@ -0,0 +1,14 @@
+from discord import Embed

```

Comment:
Rename this file better.

Reply:
Umm I think this is ok actually, idk, what are your suggestions?

Reply:
embed.py

Reply:
https://github.com/RbxAPI/Docs-Bot/pull/37/commits/eaeee9ca5392b8e6130232ab8958767b0eb76fd5